src/main/java/edu/usf/cutr/gtfsrtvalidator/background/BackgroundTask.java

```diff
@@ -86,7 +86,8 @@ public void run() {
             GtfsMetadata gtfsMetadata;
             // Holds data needed in the database under each iteration
             GtfsRtFeedIterationModel feedIteration;
-            StringBuffer consoleOutput;
+            StringBuffer consoleOutput = new StringBuffer();
+            ;
```

Comment:
unrequired semicolon

Reply:
Whoops! Not sure how that got there. Thanks, I'll fix it.